<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>98</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboModulesPath">
       <property name="toolTip">
        <string>Paths Of Modules Detected.. </string>
       </property>
       <property name="placeholderText">
        <string>Paths of Modules</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboModules">
       <property name="statusTip">
        <string>MODULE DETECTED..</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelVersion">
    <property name="geometry">
     <rect>
      <x>948</x>
      <y>870</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>V2.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>870</y>
      <width>901</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>971</width>
      <height>781</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frameClass">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelSelectedClasses">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Selected Class: </string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidgetClasses">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>321</width>
          <height>741</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchClass">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QFrame" name="frameMember">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelSelectedMemebers">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Selected Member: </string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidgetMemebers">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>321</width>
          <height>741</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchMember">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frameHelp">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelHelp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>321</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Help Information:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditHelp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>321</width>
          <height>741</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="searchHelp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>321</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
